zerver/openapi/openapi.py

```diff
@@ -95,6 +95,16 @@
     },
 }
 
+# Set of documented endpoints that doesn't have a documented response or have an
+# incomplete documented response.
+EXCLUDE_DOCUMENTED_ENDPOINTS = {
+    # This endpoint have a lot of keys and the response is not fully documented
```

Comment:
`s/have/has`.

Also, just below this we have `EXCLUDE_ENDPOINTS` list which this implementation is a clone of. We can just add `/realm:patch` to `EXCLUDE_ENDPOINTS`.

Reply:
`validate_against_openapi_schema` returns true if validation happens else false. For the second array it consists of undocumented endpoint and returns false. But for the documented endpoints we want true to make sure validation is happening. Hence another array for endpoints with only response documentation remaining so that the function will return true for them.